src/promptflow/promptflow/_cli/_pf/_flow.py

```diff
@@ -90,6 +94,12 @@ def add_parser_init_flow(subparsers):
         help="The initialized flow type.",
         default="standard",
     )
+    add_param_connection = lambda parser: parser.add_argument(  # noqa: E731
+        "--connection", type=str, help=argparse.SUPPRESS
```

Comment:
Could you include more information on the scenario in PR description? Does this only target VSC Extension or end user can also use this?

Reply:
Add the background to PR description. Currently this will be only used for AI studio.